dev-docs/source/MVPTutorial/CalculatorExample/Calculator.py

```diff
@@ -1,44 +1,28 @@
 # Mantid Repository : https://github.com/mantidproject/mantid
 #
-# Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
+# Copyright &copy; 2024 ISIS Rutherford Appleton Laboratory UKRI,
 #   NScD Oak Ridge National Laboratory, European Spallation Source,
 #   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
 # SPDX - License - Identifier: GPL - 3.0 +
 from Presenter import Presenter
 from Model import Model
 from View import View
 
```

Comment:
following previous conventions, should it be better to rename Model, View, Presenter files as model.py, view.py and presenter.py on the calculator?